edx_exams/apps/lti/tests/test_views.py

```diff
@@ -211,32 +216,14 @@ def setUp(self):
         self.course_id = 'course-v1:edx+test+f19'
         self.content_id = '11111111'
 
-        self.course_exam_config = CourseExamConfiguration.objects.create(
-            course_id=self.course_id,
-            provider=self.test_provider,
-            allow_opt_out=False
-        )
-
-        self.exam = Exam.objects.create(
-            resource_id=str(uuid.uuid4()),
+        self.exam = ExamFactory(
```

Comment:
this is just a refactor to remove more verbose data setup and replace it with factory methods.